doc/ref_model/chapters/chapter08.md

```diff
@@ -1,4 +1,320 @@
+
+In IaaS clouds, the cloud infrastructure is defined but the tenant workloads include certain needed services (such as LB, messaging); thus, the VNF/CNFs may incorporate different services with the resultant issues related to an explosion of services, their integration and management complexities. To mitigate these issues, the CNTT Reference Model must specify the common services that every Telco cloud must support and thereby require workload developers to utilise these pre-specified services.
+
+A generic Telco cloud is an hybrid multi-cloud or a Federated cloud that has deployments in large data centers, central offices or colocation facilities, and the edge. In this chapter we will discuss the characteristics of Telco Edge and hybrid multi-cloud.
+
+<a name="8.2"></a>
+## 8.2 Hybrid Multi-Cloud Architecture
+The GSMA whitepaper on "Operator Platform Concept Phase 1: Edge Cloud Computing" (January 2020) states, "Given the wide diversity of use cases that the operators will tasked to address, from healthcare to industrial IoT, it seems logical for operators to create a generic platform that can package the existing assets and capabilities (e.g., voice messaging, IP data services, billing, security, identity management, etc. ...) as well as the new ones that 5G makes available (e.g., Edge cloud, network slicing, etc.) in such a way as to create the necessary flexibility required by this new breed of enterprise customers."
+
+Cloud computing has evolved and matured since 2010 when [NIST](http://csrc.nist.gov/publications/nistpubs/800-145/SP800-145.pd) published its definition of cloud computing, with its 5 essential characteristics, 3 service models and 4 deployment models.
+
+The generic model for an enterprise cloud has to be "hybrid" with the special cases of purely private or public clouds as subsets of the generic hybrid cloud deployment model. In a hybrid cloud deployment, at least two or more distinct cloud infrastructures are inter-connected together.
+
+Cloud deployments can be created using a variety of technologies  (e.g., OpenStack, Kubernetes) and commercial technologies (e.g., VMware, AWS, Azure, etc.). A multi-cloud deployment can consist of the use of more than one technology.
+
+A generic Telco cloud is an hybrid multi-cloud. A better designation would be a federation of clouds - a federated cloud:
+   - a collection of cooperating, interoperable autonomous component clouds
+   -  the component clouds perform their local operations (internal requests) while also participating in the federation and responding to other component clouds (external requests)
+        - the component clouds are autonomous in terms of, for example, execution autonomy
```

Comment:
In case of a centralized control plane scenario the edge clouds are [not autonomus](https://wiki.openstack.org/wiki/Edge_Computing_Group/Edge_Reference_Architectures#Centralized_Control_Plane_Scenario), for example, they can not LCM their workloads.

Reply:
@CsatariGergely Thanks -- please see updated text